qa/SidechainTestFramework/sc_test_framework.py

```diff
@@ -144,7 +158,7 @@ def main(self):
             if not os.path.isdir(self.options.tmpdir):
                 os.makedirs(self.options.tmpdir)
 
-            print("Initializing test directory "+self.options.tmpdir)
+            logging.info("Initializing test directory "+self.options.tmpdir)
```

Comment:
I'm curious how it was working on Zen side with regular `print` method?
It is quite useful to see all the prints + node loges of level `ERROR` on console when running a single python test without `run_sc_tests.sh`.
Can we do something similar? So, to log to file only within the script.

Reply:
The parameters for setting log level already existed.  You can print to console by adding the --logconsolelevel=info parameter

Reply:
Thanks, it works as expected. I missed that point